### DISCUSSION TITLE
为什么会出现1080端口拒绝的情况？

### DISCUSSION THREAD

**Discussion by OLT500** [apernet/hysteria#976] [Q&A]
dial tcp 127.0.0.1:1080: connectex: No connection could be made because the target machine actively refused it.
想问下该如何解决呢？

**Comment by Gol2dragon** [comment]
请检查服务器端的防火墙是否启动了，如果启动了可以将防火墙关闭了试一下；或者将UDP 443端口打开可以通过防火墙，其它端口不用改变
